Assets/Scripts/Framework/Gameplay/Interactions/Interactable.cs

```diff
@@ -31,13 +31,27 @@ public abstract class Interactable : NetworkBehaviour
 
         private protected virtual void Awake() => BoxCollider = GetComponent<BoxCollider>();
 
+        /// <summary>
+        /// A function that returns all colliding colliders.
+        /// </summary>
+        /// <returns>An array of colliding <see cref="Collider"/>'s.</returns>
+        public Collider[] GetCollidingColliders() =>
+            Physics.OverlapBox(BoxCollider.center + transform.position, BoxCollider.size / 2, transform.rotation);
```

Comment:
Did you check to see if this is correct size?

Reply:
I did not check it with a gizmo but it should be correct this way. Everything looks normal when testing.

Reply:
Resolved.